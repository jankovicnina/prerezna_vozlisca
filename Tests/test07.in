41 65
17 3 1.94
29 4 3.87
12 0 6.15
33 20 -1.91
18 10 -0.42
20 26 5.48
27 31 6.68
27 22 9.43
35 6 0.6
0 21 3.15
7 9 8.26
16 19 1.94
28 33 2.56
40 2 6.1
39 25 0.49
6 13 0.5
31 36 2.83
25 29 10.0
18 7 6.68
5 4 4.36
7 26 0.42
14 16 9.29
8 32 0.49
15 38 5.56
4 7 5.28
30 7 3.43
21 5 4.97
33 18 2.16
10 11 6.07
31 5 6.57
23 27 3.7
7 5 5.91
15 7 0.06
3 8 1.14
5 14 7.62
20 7 -1.46
11 26 0.81
3 12 6.12
9 19 5.67
16 18 5.0
2 37 6.18
36 17 4.75
34 30 1.75
13 7 7.36
38 40 1.77
4 8 -0.34
32 24 9.79
40 25 -1.61
38 37 6.72
37 35 -0.07
13 1 -1.23
19 28 8.17
26 22 6.83
3 2 2.92
20 5 7.85
1 34 2.81
32 33 -1.56
35 30 5.56
24 15 2.09
2 17 6.67
28 31 3.86
21 39 -0.24
22 23 0.55
0 4 -0.69
30 38 4.64
