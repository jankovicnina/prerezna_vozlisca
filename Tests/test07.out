20.243518292682918
